pootle/apps/pootle_fs/files.py

```diff
@@ -205,13 +205,18 @@ def _sync_from_pootle(self):
             f.write(self.serialize())
         logger.debug("Pushed file: %s" % self.path)
 
-    def _sync_to_pootle(self, merge=False, user=None):
+    def _sync_to_pootle(self, merge=False, user=None, pootle_wins=None):
```

Comment:
yes.

It works like this
- sometimes there *is* a store_fs and a previous action was required (conflict)
- there is a store_fs but no previous action required - and then pull is called with the pootle_wins param accordingly
- there is no store_fs - pull is called with the appropriate pootle_wins param